app/scripts/metamask-controller.js

```diff
@@ -829,6 +927,49 @@ export default class MetamaskController extends EventEmitter {
     this.publicConfigStore = this.createPublicConfigStore();
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  /**
+   * Constructor helper for getting Snap permission specifications.
+   * Exists primarily to get around the circular dependencies between the Snap
+   * and Permission controllers during initialization.
+   */
+  getSnapPermissionSpecifications() {
+    // TODO:flask Probably find a better solution?
```

Comment:
We have to do this in a few places already. I'm not sure there is a better solution.

Reply:
Oh, right, these can be actions. Nevermind! We can update this comment to say that these should be actions.

Reply:
Updated to use actions as of `skunkworks@0.8.1`.